core-services/beneficiary-idgen/README.md

```
@@ -0,0 +1,44 @@
+### API Details
+
+- id/id_pool/_generate
+- id/id_pool/_dispatch
+- id/id_pool/_search
+- id/id_pool/_update
+
+## Reference document
+
+Details on every parameter and its significance are mentioned in the document: `https://digit-discuss.atlassian.net/l/c/eH501QE3`
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)